spec/ParseGeoPoint.spec.js

```diff
@@ -24,7 +24,7 @@ describe('Parse.GeoPoint testing', () => {
         });
       }
     });
-  });
+  }, 60000);
```

Comment:
Timeout on this test. I think this accommodates an old thing, since these tests don't timeout anymore but fail in a flaky way. Meaning - please remove?